### DISCUSSION TITLE
월드 출시 문제(사양)

### DISCUSSION THREAD

**Discussion by maskpop** [naverz/zepeto-studio-kor#779] [5. Q&A]
처음으로 월드 심사를 제출했는데 어떤 이유인지를 잘 모르겠어서요 ㅠㅠ
![image](https://user-images.githubusercontent.com/105036393/169475660-07e99269-fbb8-47c3-acd6-ba78e5062cf5.png)

사양을 해결하려면 어떻게 해야하는 건가요???

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 최적화의 문제이기 때문에 프로젝트 내의 오브젝트 갯수를 조절하거나, 메모리/프레임에 영향을 주는 요소를 덜어내는 방식으로 진행 부탁 드립니다.
유니티 최적화 관련 내용을 참고하여 주세요. 
https://docs.unity3d.com/kr/2019.4/Manual/MobileOptimizationPracticalScriptingOptimizations.html
